lib/tasks/populate.rake

```diff
@@ -0,0 +1,23 @@
+if Rails.env.development?
+  require 'factory_bot_rails'
+  require 'faker'
+
+  namespace :dev do
+    desc 'Sample data for local development environment'
+    task populate: 'db:migrate:reset' do
+      include FactoryBot::Syntax::Methods
```

Comment:
You can change your Gemfile to require factory_bot in test and development environments, then you will be able to use it in such tasks or in the rails console, which is pretty convenient. But, of course, do not use it in the business code